README.md

```diff
@@ -156,6 +160,54 @@ There is a know issue with session restoration on re-open though. [See GitHub is
 
 ![Firefox settings](./screenshots/firefox_config.png)
 
+### FireFox Features
+
+#### Theme Settings
+
+The Doki Theme for FireFox can synchronize with your device theme settings.
+
+Never want to see light mode ever again? No problem! The *Options page* now allows you to choose
+between `light`, `dark`, `device`, `druthers`, or `all` *(all themes)*. So take your time and
+browse.
+
+**Match Device**
+
+`device` will load only themes based on your current device theme settings *(either light or dark)*.
+In other words, your device settings determines what characters are present in the popup menu.
+
+#### Random Select *(Dark System Setting)*
+https://user-images.githubusercontent.com/23222943/146148772-c7e236ed-6be6-46b6-ac2d-aa3f1b1326ca.mp4
+
+#### Mixed Mode *(Light System Setting)*
+https://user-images.githubusercontent.com/23222943/146149663-22ad428e-0cc7-45f8-9700-1fc4c6f616d2.mp4
```

Comment:
I think it would be better to show a visual demo of the feature as well.

Reply:
Yeah, I think that is a good idea!

I was just thinking, some of the examples showcase a UI that was outdated. However, I think that keeping these all up to date with how it looks as of the current release is too much work.